client/src/index.js

```diff
@@ -15,6 +15,11 @@ const host = slashes.concat(window.location.hostname);
 const client: ApolloClient = new ApolloClient({
   uri: host + ":8080/graphql",
   cache: new InMemoryCache(),
+  resolvers: {
+    Query: {
+      customersNameList: () => "Hui",
```

Comment:
what does this line do?

Reply:
Just deleted it. Quick question, where did you put resolvers? You wrote resolvers by Go language? Should I learn a bit about it?